### ISSUE TITLE
Implementação de postagens relacionadas

### ISSUE THREAD

**Issue by fersasil** [ccuffs/app-cc#25]
Ao fim de uma postagens. exibir as postagens relacionadas

**Comment by Dovyski** [comment]
Tu precisa de uma alteração no feed RSS do site para isso, certo?

**Comment by fersasil** [comment]
Isso, mas essa é uma feature opcional, não sei se é tão necessário assim ter

**Comment by Dovyski** [comment]
Concordo que não é necessário agora. Deixamos em aberto para trabalhar no futuro.
